packages/storefront-lwc/webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
+    .BundleAnalyzerPlugin;
+
+module.exports = {
+    plugins: [
+        new BundleAnalyzerPlugin({
+            analyzerMode: 'static',
```

Comment:
This configures the webpack bundle analyzer plugin

Reply:
cool